### ISSUE TITLE
Question text should be a heading

### ISSUE THREAD

**Issue by r-torres-csps** [CSPS-RIW/Vue_QuizGen-riw-patches#45]
I believe this would make it easier to navigate with screen readers

**Comment by r-torres-csps** [comment]
Talked to Anoop, she suggests to use a `p` tag instead
